test_scripts/AppIconsFolder/ATF_N_AppIconsAmountToRemove_larger_than_icons_amount.lua

```diff
@@ -0,0 +1,326 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+--    [GENIVI] Conditions for SDL to create and use 'AppIconsFolder' storage 
+--    [AppIconsFolder]: Value of "AppIconsAmountToRemove" param larger than actual number of icons in the folder
+-- Description:
+--    SDL behavior if AppIconsAmountToRemove param in .ini file is greater than actual number of icons in the icons folder  
+-- 1. Used preconditions:
+--      Stop SDL
+--      Set AppIconsFolder in .ini file
+--      Start SDL and HMI
+--      Connect mobile
+--      Make AppIconsFolder full
+-- 2. Performed steps:
+--      Register app
+--      Send SetAppIcon   
+-- Expected result:
+--    SDL must:
+--      remove all icons from AppIconsFolder folder;
+--      write the new icon to AppIconsFolder folder
+---------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ General Settings for configuration ]]
+local preconditions = require('user_modules/shared_testcases/commonPreconditions')
+preconditions:Connecttest_without_ExitBySDLDisconnect_WithoutOpenConnectionRegisterApp("connecttestIcons.lua")
+Test = require('user_modules/connecttestIcons')
+require('cardinalities')
+local mobile_session = require('mobile_session')
+
+--[[ Required Shared Libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+require('user_modules/AppTypes')
+
+--[[ Local variables ]]
+local pathToAppFolder
+local file
+local fileContent
+local status = true
+local fileContentUpdated
+local SDLini = config.pathToSDL .. tostring("smartDeviceLink.ini")
+local RAIParameters = config.application1.registerAppInterfaceParams
+local applicationFileToCheck = config.pathToSDL .. tostring("Icons/" .. RAIParameters.appID)
```

Comment:
@aarotar 
As I can see, is used hardcoded names for folder and files. 
Could you avoid using them in this script?